activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -61,7 +61,7 @@ def test_schema_dump
 
   def test_schema_dump_uses_force_cascade_on_create_table
     output = dump_table_schema "authors"
-    assert_match %r{create_table "authors", force: :cascade}, output
+    assert_match %r{create_table "authors", .*?force: :cascade}, output
```

Comment:
In my application migrations are deleted and the schema is the source of truth, if the schema is version dependent we will have the same problem we had with non-versioned migrations. But I'm fine to change to structure.sql if I need a real snapshot of my database in the time that the migration was run.